types/src/v0/v0_1/state.rs

```diff
@@ -1,14 +1,12 @@
-use crate::{Header, ResolvableChainConfig};
-
-use super::{FeeAccount, FeeAmount};
-
+use super::{FeeAccount, FeeAmount, Header};
```

Comment:
oh the issue is here. It is importing v01 Header instead of that versioned header